lib/test.js

```diff
@@ -52,11 +52,22 @@ function Test (name_, opts_, cb_) {
     this.pendingCount = 0;
     this._skip = args.opts.skip || false;
     this._timeout = args.opts.timeout;
-    this._objectPrintDepth = args.opts.objectPrintDepth || 5;
     this._plan = undefined;
     this._cb = args.cb;
     this._progeny = [];
     this._ok = true;
+    var depthEnvVar = 'NODE_TAPE_OBJECT_PRINT_DEPTH';
+    if (args.opts.objectPrintDepth) {
+        this._objectPrintDepth = args.opts.objectPrintDepth
+    } else if (process.env[depthEnvVar]) {
```

Comment:
instead of storing the key name in a variable, and extracting it multiple times, can we just extract the value into a variable right off the bat?